test/e2e/memcached_test.go

```diff
@@ -113,6 +117,12 @@ func TestMemcached(t *testing.T) {
 		t.Fatalf("error: %v\nCommand Output: %s\n", err, string(cmdOut))
 	}
 
+	// Set replicas to 2 to test leader election. In production, this should almost always be set to 1.
```

Comment:
> should almost always be set to 1

Can you explain that part, almost always? :) Thanks!

Reply:
You would only want this to be >1 if you want a hot spare operator process running all the time. I can't think of a reason somebody would want that, but maybe a use case exists. Think I should elaborate in the in-line comment?